### PR TITLE
test

### PR DESCRIPTION
**Pull request by youngliu323**

### CODE REVIEW

File: HideAndSeek.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ function publishReview() {
     var button = reviewMenu.querySelector('button');
 
     button.onclick = function() {
-      var author = document.querySelector('.author').innerText;
+      var author = document.querySelector('.header-nav-current-user').querySelector('strong').innerText
```
Comment (kyleroush) [inline HideAndSeek.js]: test

File: HideAndSeek.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ function publishReview() {
     var button = reviewMenu.querySelector('button');
 
     button.onclick = function() {
-      var author = document.querySelector('.author').innerText;
+      var author = document.querySelector('.header-nav-current-user').querySelector('strong').innerText;
       var files = loadData()[getPullRequestId()]['files'];
       //check button and then check if they want to completes
       var meta = { author: files };
-      var rawMeta = metaData(localStorageKey, meta)
+      var rawMeta = metaData(localStorageKey, meta);
```
Comment (kyleroush) [inline HideAndSeek.js]: jhgfjhg

File: HideAndSeek.js (hunk 1 above)
Comment (kyleroush) [reply HideAndSeek.js]: hi

File: HideAndSeek.js (hunk 2 above)
Comment (kyleroush) [reply HideAndSeek.js]: i think this is garbage

Comment (kyleroush) [reply HideAndSeek.js]: asdf

Comment (kyleroush) [reply HideAndSeek.js]: asdfddfdf

Comment (kyleroush) [reply HideAndSeek.js]: kdkd

Comment (PseudoFaux) [reply HideAndSeek.js]: garbage

Comment (kyleroush) [reply HideAndSeek.js]: asdf

Comment (kyleroush) [reply HideAndSeek.js]: party_dumpster_fire

Comment (kyleroush) [reply HideAndSeek.js]: test

**Pull request closed without merging**